spec.emu

```diff
@@ -269,7 +268,7 @@
 
     <dt><dfn id="sec-terms-and-definitions-colun">column</dfn></dt>
     <dd>
-      <p>zero-based indexed offset within a line of the generated code, computed as UTF-16 code units for JavaScript and CSS source maps, and as byte indexes in the binary content (represented as a single line) for WebAssembly source maps.</p>
+      <p>zero-based indexed offset within a line of the generated code, computed as UTF-16 code units for JavaScript and CSS source maps, and as byte indeces in the binary content (represented as a single line) for WebAssembly source maps.</p>
```

Comment:
`indices`? xD

Reply:
The rule with linters is: do not fight back, accept it and don't ask questions :P

(Also, in JS `Object.getOwnPropertyNames(RegExp.prototype)` has a property called `hasIndeces` 😉)